.github/workflows/release.yml

```diff
@@ -0,0 +1,63 @@
+---
+name: Release and Publish
+
+on:
+  push:
+    tags:
+      - "v*.*.*"
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Setup python 🐍
+        uses: actions/setup-python@v4
+        with:
+          python-version: "3.10"
+      - name: Install dependencies 👽
+        run: python -m pip install ansible-core yq rst-to-myst
+
+      - name: Verify release 🗒️
+        env:
+          TAG_NAME: ${{ github.ref_name }}
+        run: |
+          echo "::warning::TODO: confirm tests pass for this commit"
+          tag_version="${TAG_NAME##*v}"
+          version="$(yq -r .version galaxy.yml)"
+          if [[ ${tag_version} != ${version} ]]; then
```

Comment:
We have extra "v" in tag, so `${tag_version} != v${version}`.

Seems in galaxy.yml, we will still have version without "v" - like here https://github.com/ansible-collections/azure/blob/v1.14.0/galaxy.yml.

Reply:
You're right. I'm actually handling that on line 26 by removing the "v" (and everything preceding it) from the `TAG_NAME` environment variable: `tag_version="${TAG_NAME##*v}"`.